config/config.go

```diff
@@ -477,6 +479,7 @@ var defaultConf = Config{
 	SplitRegionMaxNum:            1000,
 	RepairMode:                   false,
 	RepairTableList:              []string{},
+	MaxConnections:               151,
```

Comment:
Why `151`?

Reply:
It's 151 in our original code and in MySQL.

Reply:
The reason is historical. When the apache default was 150, and MySQL was 100, there were cases where web pages would contain too many connection errors. MySQL raised its limit to Apache's +1. It does not necessarily need to be followed literally.